src/shared/ui/Input/Input.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { FC } from "react";
+
+import cn from "classnames";
+
+import cls from "./Input.module.scss";
+
+export type InputProps = Omit<
+  React.InputHTMLAttributes<HTMLInputElement>,
+  "onChange"
+> & {
+  value: string;
+  onChange: (value: string) => void;
+};
+export const Input: FC<InputProps> = (props) => {
+  const { className, value, onChange, disabled, ...otherProps } = props;
```

Comment:
почему бы не делать раскрытие пропсов прямо в описании аргументов функции?
```suggestion
export const Input: FC<InputProps> = ({ className, value, onChange, disabled, ...otherProps }) => {
```

некрит